setup.py

```diff
@@ -89,7 +89,7 @@ def txt_read(*paths):
 # requirements
 setup_args['setup_requires'] =\
     [
-        "setuptools==19.2",
+        "setuptools>=19.2",
```

Comment:
The only reason of the '==' is in case of freezing with PyInstaller, otherwise the version can be dropped:
https://github.com/pyinstaller/pyinstaller/issues/1773
Although the PyInstaller ticket is closed, I had to revert to such a version to freeze on OS X. 

@jreadey : have you been able to freeze on OS X? Did you need to downgrade to `setuptools` 19.2?
However, since this bit of info is now in the [wiki](https://github.com/HDFGroup/hdf-compass/wiki/Notes-on-freezing-with-Mac-OS-X#hdf-compass-freezing), I would actually propose to drop the version requirement.